src/app/create-event/EventTextarea.tsx

```diff
@@ -0,0 +1,56 @@
+import cx from 'classnames';
+import { FieldErrors } from 'react-hook-form';
+
+import { AriaLiveErrorMessage } from '../../components/AriaLiveErrorMessage';
+
+type InputProps = {
+  register: any;
+  type?: string;
+  label: string;
+  name: string;
+  placeholder?: string;
+  errors: FieldErrors;
```

Comment:
If this would be some `Record<string, string>` or whatever the FieldErrors is, you would avoid coupling it with react hook form. But fine for this PR